samples/drivers/mbox/boards/esp32c6_devkitc_hpcore.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_ULP_COPROC_ENABLED=y
+CONFIG_DEBUG_OPTIMIZATIONS=y
```

Comment:
Why `CONFIG_DEBUG_OPTMIZATIONS' needs to be enabled by default? Can it be removed and only set in sample?